apps/web/app/api/links/[linkId]/route.ts

```diff
@@ -52,8 +52,14 @@ export const PUT = withAuth(async ({ req, headers, workspace, link }) => {
 
   const body = updateLinkBodySchema.parse(await req.json());
 
+  // Add body onto existing link but maintain NewLinkProps form for processLink
   const updatedLink = {
     ...link,
+    expiresAt:
+      link.expiresAt instanceof Date
+        ? link.expiresAt.toISOString()
+        : link.expiresAt,
+    geo: link.geo as NewLinkProps["geo"],
```

Comment:
@TWilson023 we can't do this because users won't be able to update the `expiresAt` and `geo` params for their links

Reply:
Won't these be overridden by `...body` below if users update them?

Reply:
oh yeah good point!